descriptors/afni/3dTproject.json

```
@@ -52,7 +52,11 @@
       "command-line-flag": "-cenmode",
       "description": "'kill' or 'zero' or 'ntrp'. Specifies how censored time points are treated in the output dataset:* mode = zero -- put zero values in their place; output dataset is same length as input* mode = kill -- remove those time points;  output dataset is shorter than input* mode = ntrp -- censored values are replaced by interpolated  neighboring (in time) non-censored values,  before any projections, and then the  analysis proceeds without actual removal  of any time points -- this feature is to  keep the spanish inquisition happy.* the default mode is kill !!!.",
       "optional": true,
-      "value-choices": ["KILL", "ZERO", "NTRP"]
+      "value-choices": [
+        "KILL",
+        "ZERO",
+        "NTRP"
+      ]
```

From the description it seems like these should be lowercase. is that correct?

```suggestion
      "value-choices": [
        "kill",
        "zero",
        "ntrp"
      ]
```

---

Looking at the doc [web page](https://afni.nimh.nih.gov/pub/dist/doc/program_help/3dTproject.html), it looks like it should be all capitalized.

Can take a look at the command in the container shortly.

---

Both works `all-caps` and `non-caps`.
```
3dTproject -cenmode KILL -input /styx_input/0/vol0000_trans_merged_masked_despike.nii.gz -mask /styx_input/1/MNI152_T1_1mm_brain_mask_resample_resample.nii.gz -ort /styx_input/2/nuisance_regressors.1D -polort 0 -prefix desc-residuals.nii.gz
```
```
3dTproject -cenmode kill -input /styx_input/0/vol0000_trans_merged_masked_despike.nii.gz -mask /styx_input/1/MNI152_T1_1mm_brain_mask_resample_resample.nii.gz -ort /styx_input/2/nuisance_regressors.1D -polort 0 -prefix desc-residuals.nii.gz

```

---

Seems like they do a case insensitive comparison in the source code:

https://github.com/afni/afni/blob/9b2e0debe8e3020ee7d4a51694877c3c41f7a924/src/3dTproject.c#L1345-L1352

lets go with lowercase so it does not look like it screams KILL at you.